feature/networkinstance/networkinstance_test.go

```diff
@@ -85,6 +90,30 @@ func TestAugment(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		desc:     "Static route",
```

Comment:
Once you fix the API, you can fix the test cases as well. We want to cover a few cases, i.e., prefix wthout next-hops, prefix with one next-hop, and prefix with multiple next-hops. If prefix without next-hop is invalid then we should check this in the validation and add a negative test case here.

Reply:
Done!